src/Utils.php

```diff
@@ -90,4 +91,24 @@ public static function closeAllPage(Connection $connection): void
             }
         }
     }
+
+    /**
+     * @throws CommunicationException
+     * @throws Exception\EvaluationFailed
+     *
+     * @return mixed|null
```

Comment:
invalid return type. mixed already contains null. please remove the `|null`

Reply:
ah sorry, removed. Inferred from IDE tooltip of getResult 😄